packages/frontend/web/server/htmlTemplate.ts

```diff
@@ -46,6 +50,16 @@ export const htmlTemplate = ({
             )}" as="font" crossorigin>`,
     );
 
+    const generateMetaTags = (dataObject: { [key: string]: string }) => {
+        return Object.entries(dataObject)
```

Comment:
Can you add a guard here in case `CAPI.twitterData` or `CAPI.openGraphData` is undefined or null to prevent errors?